backend/docs/SETUP_COMPOSE.md

```diff
@@ -59,6 +59,13 @@ docker-compose exec web python manage.py createsuperuser
 docker-compose -f docker-compose.dev.yml down
 ```
 
+If you want to also purge your testing database, stop the containers and remove the corresponding volumes in the host
+
+```bash
+docker-compose -f docker-compose.dev.yml down --rmi all --volumes
+rm -rf /tmp/disfactory_d*
```

Comment:
@EagleC0318 did you apply this line as well? It needs both.

Reply:
@tai271828 I think so. In fact, I feel there're actually no data under `/tmp/disfactory_d*` in my local, but the new ohshown event still there when start the container again.

Reply:
Could it be cached somewhere else?

Reply:
How do you define your `DISFACTORY_PGDATA_PATH` and `DISFACTORY_PGDATA_BACKUP_PATH` in `.env`?

See here for reference:
https://github.com/OH-SHOWN/ohshown-backend/blob/5fcf3370297c761b61f3c6058b03e73586701ca5/backend/docker-compose.dev.yml#L35

Reply:
I think it's the same, didn't modify the docker/env config in my local.

Reply:
Then I have no idea XD We will see when we need it.

Reply:
Yeah, it's fine for me if you want to check in this document change first.